### DISCUSSION TITLE
Is FDR used in GSEA Benjamini–Hochberg where alpha is 0.05?

### DISCUSSION THREAD

**Discussion by BroMynn** [zqfang/GSEApy#264] [Q&A] [answered]
First of all, thank you immensely for this wonderful library. I'm a Python lover and hate the irregularities of R. It gladdens me to see this library constantly growing and implementing many bioinformatic functionalities once only available at R. Please keep up the good work!

Now, I've got someone asking how exactly the FDR value being displayed in the normal signal-to-ratio GSEA plots is calculated. According to the GSEApy documentation, I guess p-values are corrected using Benjamini–Hochberg procedure where alpha is set at 0.05 if I used unmodified GSEApy modules (I am no expert of statistics, by the way). Am I correct?

Thank you in advance!

**Comment by zqfang** [comment, marked as answer]
Hi BroMynn,

Sorry for replying late. For the FDR implementation for GSEA, it's not the same as Benjamini–Hochberg procedure. I have followed the same strategy in the R (see [here](https://github.com/zqfang/GSEApy/blob/113499941e3870960e072c631621004572f76e66/src/stats.rs#L295)), which is same as the original paper used.

Hope this helps

**Reply by BroMynn** [reply]
I see, thank you very much!
